ctags.lua

```diff
@@ -129,11 +131,47 @@ local function get_match(word, path)
 	end
 end
 
-local function open_file(path, line)
-	vis:command(string.format('open %s', path))
-	vis.win.selection:to(tonumber(line), 1)
+--[[
+- Can't test vis:command() as it will still return true if the edit command fails.
+- Can't test File.modified as the edit command can succeed if the current file is
+  modified but open in another window and this behavior is useful.
+- Instead just check the path again after trying the edit command.
+]]
+local function goto_pos(pos)
+	if pos.path ~= vis.win.file.path then
+		vis:command(string.format('e %s', pos.path))
+	end
+	if pos.path ~= vis.win.file.path then
+		return false
+	end
```

Comment:
The second check does not make sense for the same file. Could you please move it inside the first `if`, right after the command?